platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -321,7 +326,7 @@ def update_firmware(self, image_path):
 
         try:
             self.__stage_update(image_path)
-            self.__trigger_update()
+            self.__trigger_update(allow_reboot)
```

Comment:
@alexrallen in case of pending updates (attempt to update ONiE/BIOS simultaneously) it will fail - you need to relax the original validation

Reply:
Will do.